cmd/vclusterctl/cmd/app/create/pro.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/loft-sh/loftctl/v3/pkg/config"
 	"github.com/loft-sh/loftctl/v3/pkg/vcluster"
 	"github.com/loft-sh/log"
-	"github.com/loft-sh/vcluster-values/values"
+	config2 "github.com/loft-sh/vcluster/config"
```

Comment:
Wouldn't it make sense to have this called config and alias the loftctl config to "loftctlconfig"?

Reply:
Yeah I was too lazy to do this :D